packages/web-components/src/components/va-link/va-link.tsx

```diff
@@ -302,7 +302,7 @@ export class VaLink {
             aria-label={label}
             target="_blank"
           >
-            {text} <span class="screen-only">(opens in a new tab)</span>
+            {text}&nbsp;<span class="screen-only">(opens in a new tab)</span>
```

Comment:
I'm not sure when this started happening but Chromatic was flagging a diff here where this space wasn't displaying anymore. 

![Screenshot 2025-04-01 at 10 55 12 AM](https://github.com/user-attachments/assets/b6d31b38-47c0-484e-96bb-c505f3ff3603)